distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1871,6 +1871,86 @@ type Table
             new_column = column_to_parse.parse type format on_problems
             table.set new_column new_name=column_to_parse.name set_mode=Set_Mode.Update
 
+    ## GROUP Standard.Base.Conversions
+       Formats `Column`s within a `Table` using a format string,
+       `Date_Time_Formatter`, or `Column` of format strings.
+
+       Arguments:
+       - columns: The columns to format. If not specified, all columns will be
+         formatted. The columns can have different types, but all columns must
+         be compatible with any provided `format` value.
+       - format: The type-dependent format string to use to format the values.
+         If `format` is `""` or `Nothing`, .to_text is used to format the value.
+         In case of date/time columns, the format can also be a
+         `Date_Time_Formatter`. If `format` is a `Column`, it must be a text
+         column.
+       - locale: The locale in which the format should be interpreted.
+         If a `Date_Time_Formatter` is provided for `format` and the `locale` is
+         set to anything else than `Locale.default`, then that locale will
+         override the formatters locale.
+       - error_on_missing_columns: if `True` (the default) raises an error if
+         any column is missing. Otherwise, reported as a problem.
```

Comment:
```suggestion
       - error_on_missing_columns: if `True` (the default) raises an error if
         any column is missing. Otherwise, reported as a warning.
```

Reply:
Ah, no sorry, this is not correct either.

Reply:
I'd suggest to use the same phrasing as everywhere else.

```suggestion
       - error_on_missing_columns: Specifies if a missing input column should
         result in an error regardless of the `on_problems` settings. Defaults
         to `True`.
```

IF we want to change it - let's be consistent and change all the other related comments too.

Reply:
Done.

Reply:
Also changed in `parse`, where I got this from originally :)